dietpi/dietpi-software

```diff
@@ -12318,6 +12325,27 @@ _EOF_
 			Download_Install "$(curl -sSfL 'https://api.github.com/repos/rclone/rclone/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/rclone-v[^\"\/]*linux-$arch.deb\"/{print \$4}")"
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 ))
+		then
+			Banner_Installing
+		
+			# Backup existing instance
+			if [[ -d '/var/www/homer' ]]
+			then
+				G_DIETPI-NOTIFY 2 'Existing Homer instance found, backing up to /mnt/dietpi_userdata/homer_backup ...'
+				[[ -d '/mnt/dietpi_userdata/homer_backup' ]] || G_EXEC mkdir /mnt/dietpi_userdata/homer_backup
+				[[ -d '/mnt/dietpi_userdata/homer_backup/resources' ]] && G_EXEC rm -Rf /mnt/dietpi_userdata/homer_backup/{workbox-*.js,resources}
+				G_EXEC cp -a /var/www/homer/. /mnt/dietpi_userdata/homer_backup/
+				G_DIETPI-NOTIFY 2 'Removing old assets with hash in file name ...'
+				G_EXEC rm -Rf /var/www/homer/{workbox-*.js,resources}
+			fi
+		
+			# Install latest release, preserve existing config
+			Download_Install 'https://github.com/bastienwirtz/homer/releases/latest/download/homer.zip' /var/www/homer
```

Comment:
```suggestion
			# Download the latest release
			Download_Install 'https://github.com/bastienwirtz/homer/releases/latest/download/homer.zip' homer

			# Backup existing instance
			if [[ -d '/var/www/homer' ]]
			then
				G_DIETPI-NOTIFY 2 'Existing Homer instance found, backing up to /mnt/dietpi_userdata/homer_backup ...'
				[[ -d '/mnt/dietpi_userdata/homer_backup' ]] || G_EXEC rm -R /mnt/dietpi_userdata/homer_backup
				G_EXEC cp -a /var/www/homer /mnt/dietpi_userdata/homer_backup
				G_DIETPI-NOTIFY 2 'Removing old assets with hash in file name ...'
				G_EXEC rm -Rf /var/www/homer/{workbox-*.js,resources}
			fi
		
			# Install latest release, preserve existing config
			G_EXEC cp -a homer/. /var/www/homer/
			G_EXEC rm -R homer
```

Reply:
@Joulinar 
Here the alternative suggested above.

Reply:
ah yeah that seems to be more logic 👍